modin/pandas/test/test_series.py

```diff
@@ -1432,7 +1429,17 @@ def test_dt():
     df_equals(modin_series.dt.seconds, pandas_series.dt.seconds)
     df_equals(modin_series.dt.microseconds, pandas_series.dt.microseconds)
     df_equals(modin_series.dt.nanoseconds, pandas_series.dt.nanoseconds)
-    df_equals(modin_series.dt.components, pandas_series.dt.components)
+
+    modin_components_res = modin_series.dt.components
+    pandas_components_res = pandas_series.dt.components
+    df_equals(modin_components_res, pandas_components_res)
+    assert_array_equal(
+        pandas_components_res.columns.to_numpy(),
+        modin_components_res.columns.to_numpy(),
+    )
+    assert_array_equal(
+        pandas_components_res.index.to_numpy(), modin_components_res.index.to_numpy()
+    )
```

Comment:
since fix for `df_equals` were merged and now we're checking index/columns inside it correctly, I think we can remove that check

Reply:
removed